### PR TITLE
SEQNG-680: Configuration for GHOST.

### PR DESCRIPTION
**Pull request by sraaphorst**
As per @jluhrs' comment, this is actually the work needed for 680 and not 676.

Thus, I have renamed the PR and will close the other once we get it merged.

### CODE REVIEW

File: modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/WebServerLauncher.scala [hunk 1]
```diff
@@ -180,7 +180,8 @@ object WebServerLauncher extends StreamApp[IO] with LogInitialization with Seqex
         c     <- config
         site  <- IO.pure(c.require[Site]("seqexec-engine.site"))
         giapi <- SeqexecEngine.giapiConnection.run(c)
-        seqc  <- SeqexecEngine.seqexecConfiguration(giapi).run(c)
+        // TODO: Pretty sure thhat this is wrong.
+        seqc  <- SeqexecEngine.seqexecConfiguration(giapi, giapi).run(c)
```
Comment (jluhrs) [inline modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/WebServerLauncher.scala]: @cquiroz Is this right ?

Comment (cquiroz) [reply modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/WebServerLauncher.scala]: it is wrong. this implies the gmp of GPI and GHOST are at the same host.
We need to have a `giapiGpi` and `giapiGhost`

**Comment by sraaphorst** [comment]
@cquiroz 😊I did now and realized I missed a couple things. Now it comes up fine.

**Comment by sraaphorst** [comment]
Finally! 🎉💯🌟

**Pull request merged**